src/current/v25.2/create-policy.md

```diff
@@ -53,20 +53,25 @@ The `USING` and `WITH CHECK` expressions can reference table columns and use ses
-| `DELETE`                            | ✓                                                   | —                                                | —                                                 | —                                                     | ✓                                             |
-| `INSERT ... ON CONFLICT DO UPDATE`  | ✓                                                   | —                                                | ✓                                                 | ✓                                                     | —                                             |
-| `UPSERT`                            | ✓                                                   | —                                                | ✓                                                 | ✓                                                     | —                                             |
+| Command / clause pattern             | `SELECT` policy - `USING` (row that already exists) | `SELECT` policy - `USING` (row being added) | `INSERT` policy - `WITH CHECK` (row being added) | `UPDATE` policy - `USING` (row before the change) | `UPDATE` policy - `WITH CHECK` (row after the change) | `DELETE` policy - `USING` (row to be removed) |
+|--------------------------------------|-----------------------------------------------------|---------------------------------------------|--------------------------------------------------|---------------------------------------------------|-------------------------------------------------------|-----------------------------------------------|
+| `SELECT`                             | ✓                                                   | —                                           | —                                                | —                                                 | —                                                     | —                                             |
+| `SELECT ... FOR UPDATE / FOR SHARE`  | ✓                                                   | —                                           | —                                                | ✓                                                 | —                                                     | —                                             |
+| `INSERT`                             | —                                                   | ✓                                           | ✓                                                | —                                                 | —                                                     | —                                             |
+| `INSERT ... RETURNING`               | —                                                   | ✓(a)                                        | ✓                                                | —                                                 | —                                                     | —                                             |
```

Comment:
For the `✓(a)` cases, if the policy is violated the statement will fail. So, it's kind of like `✓(b)` except that its conditionally applied if any columns are referenced in the SET, WHERE or RETURNING clause.